clib/src/utils.rs

```diff
@@ -104,6 +104,10 @@ pub(crate) fn convert_config_event(event: u16, params: u32) -> Result<ChannelEve
             let layers = convert_layers(params);
             ChannelConfigEvent::SetLayerCount(layers)
         }
+        XSYNTH_CONFIG_SETSOUNDFONT => {
```

Comment:
What even is that lol
How and why would you even pass a soundfont list as a u32

Reply:
Not sure, going to fix it now